ast/tests/build-arg.ast.json

```diff
@@ -1,4 +1,8 @@
 {
+  "Version": {
+    "major": 0,
+    "minor": 0
+  },
```

Comment:
I think it's better if this is left out when it's not set. It's less error prone when the JSON AST matches exactly to what is actually visible in text form.